### PR TITLE
docs: use simpler unpkg setup in examples

### PR DESCRIPTION
**Pull request by abeforgit**
-------

## Overview

Recommend the use of the `renderEditor` function in the unpkg setup. A single entrypoint will
be easier to maintain and it's also easier to use.

[rendered](https://github.com/lblod/frontend-embeddable-notule-editor/blob/docs/unpkg/README.md)
### How to test/reproduce

check the fiddle
### Checks PR readiness
- [ ] changelog

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -100,141 +73,63 @@ Previously we suggested using prebuilt versions of the packages hosted on a `lbl
...
-```
+See below for everything you can do with the editorElement.
 
-For a complete version of this example, checkout this file: [public/test.html](public/test.html). It also includes another button that inserts a template in the editor to showcase the plugins.
 
 ## Editor API
-The RDFa editor uses [the Prosemirror toolkit](https://prosemirror.net/) as a base. After the `editorElement.initEditor()` function is called you will have access to the editor methods, including the controller with `editorElement.controller`. This controller is an instance of the [SayController](https://github.com/lblod/ember-rdfa-editor/blob/d4472d2e237256d30333cfcc20ce6eea7db241f2/addon/core/say-controller.ts) class of the [ember-rdfa-editor](https://github.com/lblod/ember-rdfa-editor). 
+The RDFa editor uses [the Prosemirror toolkit](https://prosemirror.net/) as a base. After the `editorElement.initEditor()` function is called and awaited (the `renderEditor` function does this for you, but you might want to call it again at a later point to re-initialize the editor) you will have access to the editor methods, including the controller with `editorElement.controller`.
```
Comment (piemonkey) [inline README.md:127]: I'd avoid the long aside and just go for:
> After the `editorElement.initEditor()` function is called and awaited you will have access to the editor methods, including the controller with `editorElement.controller`. The `renderEditor` function does this for you, but you might want to call it again at a later point to re-initialize the editor.

File: README.md [hunk 2]
```diff
@@ -100,141 +73,63 @@ Previously we suggested using prebuilt versions of the packages hosted on a `lbl
...
 
 ## Editor API
-The RDFa editor uses [the Prosemirror toolkit](https://prosemirror.net/) as a base. After the `editorElement.initEditor()` function is called you will have access to the editor methods, including the controller with `editorElement.controller`. This controller is an instance of the [SayController](https://github.com/lblod/ember-rdfa-editor/blob/d4472d2e237256d30333cfcc20ce6eea7db241f2/addon/core/say-controller.ts) class of the [ember-rdfa-editor](https://github.com/lblod/ember-rdfa-editor). 
+The RDFa editor uses [the Prosemirror toolkit](https://prosemirror.net/) as a base. After the `editorElement.initEditor()` function is called and awaited (the `renderEditor` function does this for you, but you might want to call it again at a later point to re-initialize the editor) you will have access to the editor methods, including the controller with `editorElement.controller`. 
+This controller is an instance of the [SayController](https://github.com/lblod/ember-rdfa-editor/blob/d4472d2e237256d30333cfcc20ce6eea7db241f2/addon/core/say-controller.ts) class of the [ember-rdfa-editor](https://github.com/lblod/ember-rdfa-editor).
```
Comment (piemonkey) [inline README.md:128]: Should probably link to the file on master.

File: README.md [hunk 3]
```diff
@@ -100,141 +73,63 @@ Previously we suggested using prebuilt versions of the packages hosted on a `lbl
...
+fully initialized and ready for use.
 
 ## Basic Example: The editor in an HTML file 
 The idea is that you can have multiple HTML tags in which you can initialize an editor. We'll explain how it works and the process can be repeated if multiple editors are required. We need some HTML structure to start with, and then set-up the editor when everything has finished loading and rendering. We can also initialize the editor with some content or set it later via an event. Use something like the following HTML snippet as a base.  
-**Note**: the order of the JavaScript files matters.
 
 In this section, we will assume you are using unpkg, but using another service that serves npm modules or building and hosting the bundles yourself should work just as well. Simply link the corresponding files to the correct location for your setup.
 
-For an interactive example, refer to this [jsfiddle](https://jsfiddle.net/piemonkey/oLpv64c2/).
+For an interactive example, refer to this [jsfiddle](https://jsfiddle.net/abeforfiddle/7zugt5nv/25/).
```
Comment (piemonkey) [inline README.md:85]: I think the `/25` is the step number from the original fork. I think you can set this version as the base, removing the need for this, so meaning we can update the fiddle in the future without having to update the url here.

File: README.md [hunk 4]
```diff
@@ -58,40 +58,13 @@ editor.setHtmlContent('hello world');
...
-
-```bash
-dist
-└── assets
-    ├── @lblod/embeddable-say-editor-app.js
-    ├── @lblod/embeddable-say-editor.css
-    ├── @lblod/embeddable-say-editor.js
-    ├── vendor.css # currently empty and not needed
-    └── vendor.js
+<script src="https://unpkg.com/@lblod/embeddable-say-editor@3.1.x"></script>
```
Comment (piemonkey) [inline README.md:67]: I have a slight preference for pushing people (lightly) to just fix the major number as minor versions should be compatible and it avoids people being stuck on old versions...

**Review by piemonkey — APPROVED** [review]
Agreed it makes sense to encourage a more unified experience. I've added some minor comments.

File: README.md (hunk 4 above)
Comment (abeforgit) [reply README.md:67]: I pinned it cause at the time the 3.2.0 release had a broken npm package build

Comment (abeforgit) [reply README.md]: that's now resolved so I set it to ^3.2.1

**Pull request merged**